front/cypress.config.ts

```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress';
 
 export default defineConfig({
-  viewportWidth: 1200,
+  viewportWidth: 1201,
   viewportHeight: 800,
```

Comment:
Is the new desktop/large device breakpoint defined as anything _greater_ than 1200 then? When I look online I often see this breakpoint being suggested as "1200 or above( or minWidth 1200)". Just wondering the reasoning if we went with > 1200 rather than >= 1200 :)

Reply:
Good question! It's how it was defined in the [design specs](https://citizenlab.atlassian.net/browse/CL-1435) so I went with what they provided. But it's also a bit of an arbitrary choice, I suppose whether you go for 1200 or 1201 as the exact breaking point 🤷‍♀️